compose-builder/add-security.yml

```diff
@@ -171,6 +175,7 @@ services:
 
   kong:
     image: kong:${KONG_VERSION}
+    #user: "kong:kong" # Note Kong already running as kong user but doesn't expose kong user, so this fails
```

Comment:
from the Dockerfile of kong it actually runs as `kong:nogroup`  and they never create a group to run with: https://github.com/Kong/docker-kong/blob/master/alpine/Dockerfile#L40-L44   
They only created user but not group

Reply:
Ahh, great info.  I'll try "kong:nogroup". Thanks!

Reply:
maybe try `kong:0`?

Reply:
`nogroup` works and it is more explicit.

Reply:
awesome